AndroidPubSub/README.md

```diff
@@ -149,3 +149,52 @@ The keytool command does not allow importing an existing private key into a keys
         adb push <keystore name>.bks /data/user/0/your_app_dir_goes_here/files/<keystore name>
 
     The directory and filename used will depend on your use case.  Typically the application's files directory is in /data/user/0/<app namespace>/files/.  You may however choose to locate your keystore on removable media or another space on the filesystem.  The SDK allows for specifying the file path and name of the keystore so the choice is up to you.
+	
+## Using AWS IoT with authenticated users
+
+This sample demonstrates enabling unauthenticated users to use the AWS IoT APIs to securely publish-to and subscribe-from MQTT topics. Following steps demonstrates usage of AWS IoT with authenticated users. 
+
+1. Retrieve JWT Token upon successful sign in
+```
+// Get id token from CognitoUserSession.
+String ids = cognitoUserSession.getIdToken().getJWTToken();
+```
+
+2. Setup credentials provider.
+
+```
+ // Create a credentials provider, or use the existing provider.
+credentialsProvider = new CognitoCachingCredentialsProvider(
+                getApplicationContext(), // context
+                COGNITO_POOL_ID, // Identity Pool ID
+                MY_REGION // Region
+        );
+// Set up as a credentials provider.
+Map<String, String> logins = new HashMap<String, String>();
+// Key must be of the form "cognito-idp.<REGION>.amazonaws.com/<COGNITO_USER_POOL_ID>"
+logins.put("cognito-idp.us-east-1.amazonaws.com/us-east-1_123456XXX", cognitoUserSession.getIdToken().getJWTToken()); 
+credentialsProvider.setLogins(logins);
+```
+
+3. Create MQTT client and connect
+
+```
+// MQTT Client
+mqttManager = new AWSIotMqttManager(clientId, CUSTOMER_SPECIFIC_ENDPOINT);
```

Comment:
can we elaborate little more on what the CUSTOMER_SPECIFIC_ENDPOINT is? Is it a constant in the sample or the developer needs to specify a value here??